import-clips/run.js

```diff
@@ -60,14 +74,29 @@ const OVERLAP_TIME = 2;
 const CLIP_LENGTH = 4; 
 
 if (youtubeUrl) {
+  console.log(youtubeUrl)
   videoId = getVideoId(youtubeUrl);
   fetchVideoInfo(videoId, (err, videoInfo) => {
     if (err) throw new Error(err);
     videoDuration = videoInfo.duration;
     console.log(videoDuration);
     const clipIntervals = getClipIntervals(videoDuration, CLIP_LENGTH, OVERLAP_TIME);
-    newRows = clipIntervals.map( interval => formatRow(youtubeUrl, interval))
-    sendToDB(newRows);
+    newRows = clipIntervals.map( interval => formatRow(youtubeUrl, interval));
+    if(TESTING){
+      con.connect(function(err) {
```

Comment:
I should have done it this way to start but I was lazy and used the flask app end point. Let's make this into a method, and we can use it to update local or prod.